uint64 TimeStamp
int32 Speed
float32 AngularRate
bool Brake
